src/map/script.cpp

```diff
@@ -16613,6 +16613,21 @@ BUILDIN_FUNC(mapid2name)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+/// Returns the map ID of given map name. By [KWDev]
```

Comment:
```suggestion
/// Returns the map ID of given map name.
```
Your authorship will be seen in the git log. Please dont take it personal.